pkg/kt/snmp.go

```diff
@@ -133,6 +133,19 @@ type V3SNMPConfig struct {
 	origStr                  string
 }
 
+type EAPIConfig struct {
+	Host      string `yaml:"host"`
+	Username  string `yaml:"username"`
+	Password  string `yaml:"password"`
+	Transport string `yaml:"transport"`
+	Port      int    `yaml:"port"`
+}
```

Comment:
will this section accept env vars and aws secrets?

Reply:
will add.